app/controllers/import/prospects_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Import
+  class ProspectsController < ApplicationController
+    include ActionView::Helpers::TextHelper
+
+    def create
+      prospects = ProspectApiWrapper.instance.get_prospects
```

Comment:
`prospects` is misleading here. Consider `prospects_attributes` (or return objects, e.g. `ApiProspects`)